DeAceroDLabs/marketplace-frontend
# [DMP-597] Use new Roobyn inventory endpoints

## What is done

[[DMP-597] Actualizar endpoints de Roobyn inventory](https://deacero.atlassian.net/browse/DMP-597)

## How is done

1. Modify the object types that are expected in the response of the endpoint calls as well as the requests.
2. Modify the files that were using the endpoints to use new signatures.
3. Modify tests.

## How to test

1. On the backend project move the git branch to `feature/DMP-545-roobyn-migration`.
2. Run `npm start`

1. On the frontend project run `npm start`.
3. Everything should work as before.

NOTE: Right now the Backend is returning dummy data so we will not be seeing data as before.
